config/jobs-chromeos.yaml

```
@@ -440,6 +440,19 @@ jobs:
       branch:
         - for-kernelci
 
+  kselftest-error-logs:
+    template: kselftest.jinja2
+    kind: test
+    params:
+      nfsroot: 'http://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/20240313.0/{debarch}'
+      collections: devices/error_logs
+      job_timeout: 10
+    rules:
+      tree:
+        - collabora-next
+      branch:
+        - for-kernelci
```

Hi, Could you please add `kcidb_test_suite` property for KCIDB test names mapping:
```suggestion
  kselftest-error-logs:
    template: kselftest.jinja2
    kind: test
    params:
      nfsroot: 'http://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/20240313.0/{debarch}'
      collections: devices/error_logs
      job_timeout: 10
    rules:
      tree:
        - collabora-next
      branch:
        - for-kernelci
    kcidb_test_suite: kselftest.error_logs 
```